types/picomatch/lib/scan.d.ts

```diff
@@ -0,0 +1,44 @@
+interface Options {
```

Comment:
It would be nice to bundle these auxiliary types into a `namespace scan` so they can be exported alongside the function.